src/data/git-quiz.ts

```diff
@@ -630,6 +630,17 @@ const gitQuiz = [
     Explanation:
       "The 'git help' command will display help information on common Git commands. The 'git help git' command will provide a complete overview of the system.",
       Link: "https://www.freecodecamp.org/news/best-git-tutorial/"
+},
+  {
+    Question: "When using the 'git stash' command, what is one of the major disadvantages of using it?",
+    Answer: 
+      "cluster and confusion",
```

Comment:
this is the only choice between distractors and answers that is not a full sentence, answers and distractors for a single question should all have the same style please

Reply:
Changed the answer to a complete sentence.